articles/design/creating-invite-only-applications.md

```diff
@@ -1,118 +1,137 @@
 ---
-description: How to customize the signup process for an invite-only application with Auth0
+description: Describes how to customize the signup process for an invite-only application with Auth0.
 toc: true
 crews: crew-2
 topics:
   - design
-contentType: tutorial
-useCase: strategize
+contentType: how-to
+useCase: invite-only
 ---
-# Invite-Only Applications
+# Implement Invite-Only Applications
 
-Many SaaS apps allow self-service provisioning, where users can register themselves and begin using the app. Other types of apps, however, do not allow such signups. Instead, the customer (typically an organization of some type) pay upfront for a number of users, and only the end user with the appropriate credentials may sign up and access the app. In such cases, you can use an invite-only workflow for authorization purposes.
+User invites are typically used when users do not self-provision (signup) for access to an application. Instead, an administrator creates the user and then invites the user to complete the signup process by creating a password for their account. 
 
-## Example Scenario: ExampleCo
+You can invite users to access your applications using Change Password tickets in Auth0. The authorization flow consists of the following steps:
 
-In this tutorial, we will work through a sample setup for the fictional company, ExampleCo. 
+1. User receives an email inviting them to register.
+2. User follows a link to the **Set Up Password** page. 
+3. User creates and verifies a password.
+4. User signs in.
+5. User has access to the target application.
 
-ExampleCo is a multi-tenant SaaS solution offering cloud-based analytics. Customers purchasing licenses send ExampleCo lists of users whom they want to access the application.
+A user invitation is not always a transactional email and therefore is not an email Auth0 generates for you. Instead, you can create a link via API to construct the invitation email to be sent with your existing email service provider.
```

Comment:
A user invitation is in essence a change password link repurposed as an invitation.  There are two common approaches for this. 

**Sending an Auth0 change password email**
This can be done by sending the [change password email](https://auth0.com/docs/email/custom#change-password-confirmation-email) and [customizing the template](https://auth0.com/docs/email/templates)

**Creating a password change ticket**